crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -317,12 +294,14 @@ impl SequencerBlockData {
                 if let Some(action) = action.as_sequence() {
                     // TODO(https://github.com/astriaorg/astria/issues/318): intern
                     // these namespaces so they don't get rebuild on every iteration.
-                    rollup_data
-                        .entry(ChainId(action.chain_id.clone()))
-                        .and_modify(|data: &mut Vec<Vec<u8>>| {
-                            data.push(action.data.clone());
-                        })
-                        .or_insert_with(|| vec![action.data.clone()]);
+                    // FIXME(https://): change sequence actions to carry string chain IDs
+                    let chain_id = String::from_utf8_lossy(&action.chain_id);
```

Comment:
is it better to us `from_utf8_lossy` over `from_utf8`?

Reply:
The problem is that we don't know what's inside `action.chain_id`. Right now we define it as `bytes` (in the protobuf definition), so it could be bytes that are not utf8.